src/gui/guiFormSpecMenu.cpp

```diff
@@ -429,36 +430,31 @@ void GUIFormSpecMenu::parseList(parserData* data, const std::string &element)
 			3
 		);
 
-		v2s32 pos;
-		core::rect<s32> rect;
+		v2f32 slot_spacing = data->real_coordinates ?
+				v2f32(imgsize.X * 1.25f, imgsize.Y * 1.25f) : spacing;
 
-		if (data->real_coordinates) {
-			pos = getRealCoordinateBasePos(v_pos);
-			rect = core::rect<s32>(pos.X, pos.Y,
-					pos.X + (geom.X - 1) * (imgsize.X * 1.25) + imgsize.X,
-					pos.Y + (geom.Y - 1) * (imgsize.Y * 1.25) + imgsize.Y);
-		} else {
-			pos = getElementBasePos(&v_pos);
-			rect = core::rect<s32>(pos.X, pos.Y,
-					pos.X + (geom.X - 1) * spacing.X + imgsize.X,
-					pos.Y + (geom.Y - 1) * spacing.Y + imgsize.Y);
-		}
+		v2s32 pos = data->real_coordinates ? getRealCoordinateBasePos(v_pos)
+				: getElementBasePos(&v_pos);
 
-		gui::IGUIElement *e = new gui::IGUIElement(EGUIET_ELEMENT, Environment,
-				this, spec.fid, rect);
+		core::rect<s32> rect = core::rect<s32>(pos.X, pos.Y,
+				pos.X + (geom.X - 1) * slot_spacing.X + imgsize.X,
+				pos.Y + (geom.Y - 1) * slot_spacing.Y + imgsize.Y);
 
-		// the element the list is bound to should not block mouse-clicks
-		e->setVisible(false);
+		GUIInventoryList *e = new GUIInventoryList(Environment, this, spec.fid,
+				rect, m_invmgr, loc, listname, geom, start_i, imgsize, slot_spacing,
+				this, data->inventorylist_options.slotbg_n,
+				data->inventorylist_options.slotbg_h,
+				data->inventorylist_options.slotborder,
+				data->inventorylist_options.slotbordercolor, m_font);
```

Comment:
Link target placeholder

Reply:
?
Is this the same as the next comment?